### PR TITLE
Adds standalone plugin install from ZIP files

### PR DESCRIPTION
**Pull request by aetter**
@salamanderrex

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by alolita** [comment]
Reviewed. LGTM @aetter

File: docs/install/plugins.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+See [Version history](../../../#version-history) for the versions of Elasticsearch that Open Distro for Elasticsearch supports. You must have the exact compatible version installed (e.g. 6.6.2 and not 6.6.1). To get a list of available Elasticsearch versions on CentOS 7 and Amazon Linux 2:
+
+```bash
+sudo yum --showduplicates list elasticsearch
+```
+
+Then you can specify the version you need:
+
+```bash
+sudo yum install elasticsearch-6.6.2-1
```
Comment (salamanderrex) [inline docs/install/plugins.md]: In order to this, we need to specify the RPM repo location. Should we mention it here?

https://www.elastic.co/guide/en/elasticsearch/reference/6.3/rpm.html#rpm-repo

Also, I think the very ending `-1` is not necessary.

File: docs/install/plugins.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+# Standalone plugin install
+
+If you don't want to use the all-in-one Open Distro for Elasticsearch Docker images or RPM packages, you can install the Security, Alerting, SQL, and Performance Analyzer plugins on a compatible Elasticsearch cluster just like any other Elasticsearch plugin. Navigate to the Elasticsearch home directory (likely `/usr/share/elasticsearch`) and run:
+
+```bash
+sudo bin/elasticsearch-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-security/opendistro_security-0.8.0.0.zip
+sudo bin/elasticsearch-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-alerting/opendistro_alerting-0.8.0.0.zip
+sudo bin/elasticsearch-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-sql/opendistro_sql-0.8.0.0.zip
+sudo bin/elasticsearch-plugin install https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/performance-analyzer/opendistro_performance_analyzer-0.8.0.0.zip
```
Comment (salamanderrex) [inline docs/install/plugins.md]: Actually, there is no good way to get minor version for plugins. For example, 0.7.1's security plugin is 0.7.0.1. But there is no reference to it.

File: docs/install/plugins.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+sudo yum --showduplicates list elasticsearch
+```
+
+Then you can specify the version you need:
+
+```bash
+sudo yum install elasticsearch-6.6.2-1
+```
+
+After installing the Security plugin, you can run `sudo sh /usr/share/elasticsearch/plugins/opendistro_security/tools/install_demo_configuration.sh` to quickly get started or configure it manually for production workloads.
```
Comment (salamanderrex) [inline docs/install/plugins.md]: Should we mention that `get quickly get started with demo certs` ?

File: docs/install/plugins.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Standalone Plugin Install
+parent: Install and Configure
+nav_order: 98
+---
+
+# Standalone plugin install
+
+If you don't want to use the all-in-one Open Distro for Elasticsearch Docker images or RPM packages, you can install the Security, Alerting, SQL, and Performance Analyzer plugins on a compatible Elasticsearch cluster just like any other Elasticsearch plugin. Navigate to the Elasticsearch home directory (likely `/usr/share/elasticsearch`) and run:
```
Comment (salamanderrex) [inline docs/install/plugins.md]: What about kibana? If someone install security plugin, I think security kibana is also needed?

File: docs/install/plugins.md (hunk 1 above)
Comment (aetter) [reply docs/install/plugins.md]: I'd rather operate under the assumption that people have an existing cluster and not get too far into the weeds on installing ES on your own.

I'll definitely remove the unnecessary -1, though. It works, too, but is ugly -- slipped in during my copy and pasting.

File: docs/install/plugins.md (hunk 3 above)
Comment (aetter) [reply docs/install/plugins.md]: Yep, will do.

File: docs/install/plugins.md (hunk 4 above)
Comment (aetter) [reply docs/install/plugins.md]: Do we have the Kibana plugins available, too? I'm happy to add them, but the CloudFront URL is locked down such that you can't just browse for all available files.

File: docs/install/plugins.md (hunk 2 above)
Comment (aetter) [reply docs/install/plugins.md]: Any clarity change needed here? Should we just number everything X.Y.Z on CloudFront?

**Pull request merged**